### DISCUSSION TITLE
How set Boot back to (None)

### DISCUSSION THREAD

**Discussion by Bee91303** [picocomputer/community#108] [Q&A] [answered]
I'm trying to update the basic Rom.  I'm standing on the rug I'm trying to lift.  I put this in discord as well.  When I type Remove BASIC , i get "Can't remove Boot ROM"  I have "set boot basic".  What are the steps to set Boot back to None or something else?

Thx

**Comment by Bee91303** [comment, marked as answer]
I'd like a more elegant solution but the quick answer is - use the nuke file

https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html#resetting-flash-memory

Reflash the RIA and VGA

When doing this with just the RIA he VGA was always "unknown"

I hope this helps someone else.  The original question stands.

Thx

**Comment by rumbledethumps** [comment]
"help set boot" has what you need.

**Reply by Bee91303** [reply]
Thank you - I'm really enjoying this so far, just like my Atari days
